### DISCUSSION TITLE
Course: Build a Web3 App with Solidity + Ethereum Smart Contracts by buildspace

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#6] [General]
@buildspace is a community to bring you from web2 to web3 experiences. They will kick-off the course [Build a Web3 App with Solidity + Ethereum Smart Contracts](https://zip.sc/cpiYg) on **Sat, Nov 20 at 3:00PM Eastern**.

<img width="1530" alt="buildspace" src="https://user-images.githubusercontent.com/131172/141944535-de69379a-9dfd-4ccd-9323-1dedbfc65cd8.png">

**Comment by samnang** [comment]
Are there any members working on buildspace Web3 project this week? Ping me if you want to discuss what we have done so far with the project.

**Reply by Uysim** [reply]
I don't have time last week. So I just start with it this week. Just my first smart contract with console.

**Reply by ks-nireak** [reply]
This course is somehow a 2 weeks period course. So it might end this week Saturday. Better be quick.

**Reply by samnang** [reply]
I have done it last week. I already got their NFT:

![image](https://user-images.githubusercontent.com/131172/144535322-eb1cc9f2-b0d8-4eff-b927-ab628b052ff4.png)

https://opensea.io/assets/matic/0x3cd266509d127d0eac42f4474f57d0526804b44e/4624/

**Reply by Uysim** [reply]
Do you think they still open for learning after the deadline?

**Reply by samnang** [reply]
I don't know, but they should be able to re-join their next batch.

**Comment by Uysim** [comment]
I just completed Build a Web3 App with Solidity + Ethereum Smart Contracts.
https://waveportal-starter-project.uysimty.repl.co/
Really enjoy!
